src/stories/Introduction.stories.mdx

```diff
@@ -0,0 +1,7 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Introduction" />
+
+## Hello world
+
+Todo
```

Comment:
Is this for a future "doc" PR?

Reply:
Yes. Since it's going to be included by source we will need some introductory material for people to consume it easily. Depending on what's necessary we may be able to reference `README.md` from this file as well.